README.md

```diff
@@ -270,6 +271,10 @@ APIs
 - [Withings](http://oauth.withings.com/api) - Access to data from Withings activity trackers and body measures.
 - [USDA API](https://ndb.nal.usda.gov/ndb/doc/index) - United States Department of Agriculture Open Source API for gathering nutritional data, and food reports.
 
+### Geocoding
```

Comment:
- We usually reserve creating a category if there is > 1 APIs under that category. Let's put it in the Miscellaneous category for now.
- Since this API has tiered pricing, let's follow the PR guidelines for paid APIs.

Reply:
@n0shake Thanks for taking at look at this. I've updated the PR accordingly.